boards.txt

```diff
@@ -725,3 +772,31 @@ Maple.menu.opt.o3lto.build.flags.ldspecs=-flto
 Maple.menu.opt.ogstd=Debug (-g)
 Maple.menu.opt.ogstd.build.flags.optimize=-g -Og
 Maple.menu.opt.ogstd.build.flags.ldspecs=
+
+RemRam.menu.opt.osstd=Smallest (-Os default)
+RemRam.menu.opt.osstd.build.flags.optimize=-Os
```

Comment:
Remove those 2 lines, they are useless as this is the default in platform.txt
```
RemRam.menu.opt.osstd.build.flags.optimize=-Os
RemRam.menu.opt.osstd.build.flags.ldspecs=
```

Reply:
You're sure about that? Using Eclipse / Sloeber won't show me the "-Os" Option without LTO if I remove those two lines.

Reply:
Ah okay, I got confused with the line numbering. Now I understand.